log_outgoing_requests/validators.py

```diff
@@ -0,0 +1,54 @@
+from typing import TYPE_CHECKING, Any, Iterable, Optional
+
+if TYPE_CHECKING:  # pragma: nocover
+    from .models import OutgoingRequestsLogConfig
+
+
+def _validate_content_length(
+    content_length: Any, config: "OutgoingRequestsLogConfig"
+) -> bool:
+    max_content_length: Optional[int] = config.max_content_length
```

Comment:
the type should be properly inferred when you use the `django-stubs` package :thinking: